yamllint/cli.py

```diff
@@ -140,6 +140,18 @@ def show_problems(problems, file, args_format, no_warn):
 
     return max_level
 
+def find_project_config(path='.'):
+
+    for filename in [ '.yamllint', '.yamllint.yaml', '.yamllint.yml' ]:
```

Comment:
You can remove the empty line, and also use parentheses rather than brackets, like in the test file.